app/views/finders/_search_results.html.erb

```diff
@@ -0,0 +1,80 @@
+<ul class="finder-results" data-module="track-click">
+  <% if local_assigns[:display_grouped_results] %>
+    <% local_assigns[:grouped_documents].each do |group| %>
+      <li class="filtered-results__group">
+        <% if group[:facet_name] %>
+          <h2 class="filtered-results__facet-heading"><%= group[:facet_name] %></h2>
+        <% end %>
+        <ul class="finder-results">
+          <% group[:documents].each do |document| %>
+            <% result = document[:document] %>
+            <li class="document">
+              <a href='<%= result[:link] %>'
+                  class="<% if result[:promoted] %>document-heading--pinned<% end %> document__link"
+                  data-track-category='navFinderLinkClicked'
+                  data-track-action='<%= finder_name %><%= result[:document_index] %>'
+                  data-track-label='<%= result[:link] %>'
+                  data-track-options='{"dimension28":"<%= result[:page_count] %>","dimension29":"<%= result[:title] %>"}'
+              ><%= result[:title] %></a>
+              <% if result[:promoted_summary] %>
+                <p><%= result[:promoted_summary] %></p>
+              <% end %>
+              <% if result[:is_historic] %>
+                <p class="historic">First published during the <%= result[:government_name] %></p>
+              <% end %>
+            </li>
+          <% end %>
+        </ul>
+      </li>
+    <% end %>
+
+  <% else %>
+    <% local_assigns[:documents].each do |document| %>
+      <% result = document[:document] %>
+
+      <li class="document<% if result[:top_result] %> document--top<% end %>">
+          <% if result[:top_result] %><p class="document__heading--top">Most relevant result</p><% end %>
+        <a href='<%= result[:link] %>'
+            class="document__link<% if result[:top_result] %> document__link--top<% end %>"
+            data-track-category='navFinderLinkClicked'
+            data-track-action='<%= finder_name %>.<%= document[:document_index] %>'
+            data-track-label='<%= result[:link] %>'
+            data-track-options='{"dimension28":"<%= page_count %>","dimension29":"<%= result[:title] %>"}'
+        ><%= result[:title] %></a>
+        <% if result[:summary] %>
+          <p><%= result[:summary] %></p>
+        <% end %>
```

Comment:
Can we extract lines 35-46 and lines 11-21 to a separate template as they're almost the same?

Reply:
They do look similar, but have a number of key differences that I think would make that new template a little messy. For example, 

`data-track-action='<%= finder_name %><%= result[:document_index] %>'` 
`data-track-action='<%= finder_name %>.<%= document[:document_index] %>'`

and

`data-track-options='{"dimension28":"<%= result[:page_count] %>","dimension29":"<%= result[:title] %>"}'`
`data-track-options='{"dimension28":"<%= page_count %>","dimension29":"<%= result[:title] %>"}'`